providers/base/units/wireless/test-plan.pxu

```diff
@@ -42,9 +42,9 @@ include:
     wireless/wireless_connection_wpa_ax_nm_.*      certification-status=blocker
     wireless/wireless_connection_wpa3_ax_nm_.*     certification-status=blocker
     wireless/wireless_connection_open_ax_nm_.*     certification-status=blocker
-    wireless/wireless_connection_wpa_be_nm_interface  certification-status=blocker
-    wireless/wireless_connection_wpa3_be_nm_interface certification-status=blocker
-    wireless/wireless_connection_open_be_nm_interface certification-status=blocker
+    wireless/wireless_connection_wpa_be_nm_.*      certification-status=blocker
```

Comment:
The preferred way to do this now is using the template id ("wireless/wireless_connection_wpa_XX_nm_interface") to create the test plan instead of the regex pattern. Instead of changing this back to it's original state, please change the rest of the tests to use the template ID if you want to keep the previous order.